x-pack/plugin/ccr/qa/src/main/java/org/elasticsearch/xpack/ccr/ESCCRRestTestCase.java

```diff
@@ -229,26 +243,9 @@ protected static void verifyAutoFollowMonitoring() throws IOException {
             throw new AssertionError("error while searching", e);
         }
         assertNotNull(responseEntity);
-        int numberOfSuccessfulFollowIndices = 0;
-
-        List<?> hits = (List<?>) XContentMapValues.extractValue("hits.hits", response);
-        assertThat(hits.size(), greaterThanOrEqualTo(1));
 
-        for (int i = 0; i < hits.size(); i++) {
-            Map<?, ?> hit = (Map<?, ?>) hits.get(i);
-
-            int foundNumberOfOperationsReceived = (int) XContentMapValues.extractValue(
-                "_source.ccr_auto_follow_stats.number_of_successful_follow_indices",
-                hit
-            );
-            numberOfSuccessfulFollowIndices = Math.max(numberOfSuccessfulFollowIndices, foundNumberOfOperationsReceived);
-        }
-
-        assertThat(
-            "Unexpected number of followed indices [" + responseEntity + ']',
-            numberOfSuccessfulFollowIndices,
-            greaterThanOrEqualTo(1)
-        );
+        final Number count = (Number) XContentMapValues.extractValue("count", response);
+        assertThat("Unexpected number of followed indices [" + responseEntity + ']', count.longValue(), greaterThanOrEqualTo(1L));
```

Comment:
It asserts a doc count rather then an aggregated metric now.
Is the message still correct with change?

Reply:
Thanks for catching this, I updated the message to be
> "Expected at least 1 successfully followed index but found none, count returned [" + responseEntity + ']'